lib/commands/gesture.js

```diff
@@ -1,9 +1,28 @@
 import { errors } from 'appium-base-driver';
 import { util } from 'appium-support';
-
+import { retryInterval } from 'asyncbox';
 
 let helpers = {}, extensions = {}, commands = {};
 
+commands.click = async function (el) {
+  let application;
+  let scrollIntoView = async () => {
+    let endpoint = `/element/${el}/location_in_view`;
+    let response = await this.proxyCommand(endpoint, 'GET');
+    if (response && response.status !== 0 && response.value && response.value.indexOf("Failed to perform scroll") > -1) {
+      if (!application) {
+        application = await this.findElement(`class name`, `XCUIElementTypeApplication`);
+      }
+      let params = { direction: `down`, element : application };
+      await this.mobileScroll(params);
+      throw new Error(`could not scroll into view`);
+    }
+  };
+  await retryInterval(5, 1, scrollIntoView);
```

Comment:
No. Because we really dont have easier way to get parent of element and then determine its size and location. We need to implement separate endpoint in WDA